american-journal-of-enology-and-viticulture.csl

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>American Journal of Enology and Viticulture</title>
+    <title-short>AJEV</title-short>
+    <id>http://www.zotero.org/styles/american-journal-of-enology-and-viticulture</id>
+    <link href="http://www.zotero.org/styles/american-journal-of-enology-and-viticulture" rel="self"/>
+    <link href="http://www.zotero.org/styles/friedrich-schiller-university-jena-faculty-of-medicine" rel="template"/>
+    <link href="http://www.ajevonline.org/content/authors" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="medicine"/>
```

Comment:
@POBrien333, wine is not really a medicine ;).

Reply:
well... https://doi.org/10.4103%2F0975-3583.74259

Reply:
"All Breathlessness In Mitral Stenosis Is Not Pulmonary Edema- A Case of Mitral Stenosis with Cardio Respiratory Arrest" ? Don't see the wine connection.

Reply:
oops copied the DOI from PMC. Was supposed to be this one: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3023893/

Reply:
Better, but my complaint stands :).

(I reported the DOI problem at https://support.crossref.org/hc/en-us/requests/new. It should resolve to http://old.jcdronline.org/article/145 but resolves to http://www.jcdronline.org/article/145. Note the identical article no.)